packages/twenty-docker/prod/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.8'
+name: twenty
+
+services:
+  server:
+    image: twentycrm/twenty:${TAG}
+    volumes:
+      - server-local-data:/app/.local-storage
+    ports:
+      - "3000:3000"
+    environment:
+      PORT: 3000
+      PG_DATABASE_URL: postgres://twenty:twenty@${PG_DATABASE_HOST}/default
+      SERVER_URL: ${SERVER_URL}
+      FRONT_BASE_URL: ${FRONT_BASE_URL:-$SERVER_URL}
+
+      ENABLE_DB_MIGRATIONS: true
+
+      SIGN_IN_PREFILLED: ${SIGN_IN_PREFILLED}
+      STORAGE_TYPE: local
```

Comment:
maybe we still want to let people overwrite that in env no? And local is the default config in env and in the app